dbms/src/Flash/Planner/PhysicalPlan.cpp

```diff
@@ -82,19 +83,32 @@ void PhysicalPlan::build(const tipb::DAGRequest * dag_request)
         });
 }
 
+void PhysicalPlan::buildTableScan(const String & executor_id, const tipb::Executor * executor)
+{
+    TiDBTableScan table_scan(executor, executor_id, dagContext());
+    if (unlikely(context.isTest()))
+        pushBack(PhysicalMockTableScan::build(context, executor_id, log, table_scan));
+    else
+        pushBack(PhysicalTableScan::build(executor_id, log, table_scan));
+    dagContext().table_scan_executor_id = executor_id;
+}
+
 void PhysicalPlan::build(const String & executor_id, const tipb::Executor * executor)
 {
     assert(executor);
     switch (executor->tp())
     {
     case tipb::ExecType::TypeLimit:
+        GET_METRIC(tiflash_coprocessor_executor_count, type_limit).Increment();
```

Comment:
When will we decrease the metric?

Reply:
It don't need to decrease, because of https://github.com/pingcap/tiflash/blob/35bbaf0695817034d9169c8c997846f73f5c2e69/metrics/grafana/tiflash_summary.json#L2517